lib/matplotlib/legend.py

```diff
@@ -517,8 +518,12 @@ def val_or_rc(val, rc_name):
             if not self.isaxes and loc in [0, 'best']:
                 loc = 'upper right'
 
+        type_err_message = ("loc must be string, coordinate tuple, or"
+                            f" an integer 0-10, not {loc!r}")
+
         # handle outside legends:
         self._outside_loc = None
+
```

Comment:
Could you remove this added blank line?  Otherwise, I think this is good to go.

Reply:
done.